src/content/docs/usage/pulse.md

```diff
@@ -23,3 +23,17 @@ With Bulk Restore, you can:
 * **Multi-Selection Mode:** Easily activate multi-select mode, choose the tests and suites you wish to restore, and bring them all back with a single action.
 
 ![Testomat.io - Bulk Restore](./images/New_85QaXaUI_2024-07-30.gif)
+
+## Trash Bin
+
+The **Trash Bin** feature is designed to enhance data recovery and user experience by allowing users to easily restore accidentally deleted suites or test cases. This feature ensures that valuable testing data is not permanently lost and can be quickly recovered without the need for complex restoration processes.
+
+With **Trash Bin**, you can confidently:
+
+* **Data Safety:** Minimizes the risk of permanent loss of important test data due to accidental deletion.
+
+* **User Convenience:** Provides a straightforward way to recover deleted items directly from the Tests page.
+
+* **Efficiency:** Saves time and resources by eliminating the need for external recovery methods or administrative intervention.
```

Comment:
I don't think that this 3 point actually mean anything

We can cut this to: "in case you or your teammates accidentally deleted tests or suite you can find them in a trash bin". Actually it was written in previous paragraph.